package.json

```diff
@@ -8,6 +8,13 @@
     "tap": "bin/run.js"
   },
   "main": "lib/tap.js",
+  "exports": {
+    ".": {
+      "import": "./lib/tap.mjs",
+      "default": "./lib/tap.js"
```

Comment:
This should be backwards compatible to most versions of Node.js. Any versions that don't understand package.exports will get main. All versions that supported exports support the default condition.

exporting `"./": "./"` maintains deep traversal of the module to keep this change `Semver-Minor` but you could start locking down the interface of the package if you wanted to in a future major